persistent/picklecache.py

```diff
@@ -156,18 +229,20 @@ def klass_items(self):
     def incrgc(self, ignored=None):
         """ See IPickleCache.
         """
-        target = self.target_size
+        target = self.cache_size
```

Comment:
OK, given that `cache_size` is documented, let's just remove the `target_size` attribute altogether -- the C implementation doesn't expose it.